catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# Install dependencies
+if [ -f /usr/bin/pacman ] 
+then
+  sudo pacman -S curl wget unzip git
+elif [ -f /usr/bin/emerge ]
+then
+  sudo emerge -av curl wget unzip git
+elif [ -f /usr/bin/apt ]
+then
+   sudo apt install curl wget unzip git
+elif [ -d /nix ]
+then
+  sudo nix-shell -p curl wget unzip git
+elif [ -f /usr/bin/dnf ]
+then
+  sudo dnf install curl wget unzip git
+fi
+
+# Make temp directory
+cd "$(mktemp -d)" || exit 1
+
+# Get the latest release of SFP and unzip it
+mkdir SFP 
+cd SFP || exit 1
+curl -s https://api.github.com/repos/PhantomGamers/SFP/releases/latest \
+| grep "SFP_UI-SelfContained-linux-x64.zip" \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
+unzip *.zip
+chmod +x SFP_UI
+./SFP_UI 
+
+# Install metro
+if [ -d ~/.steam ]
+then
+  mkdir -p ~/.steam/steam/skins/
+  git clone https://github.com/minischetti/metro-for-steam ~/.steam/steam/skins/catppuccin
+  export METRO="$HOME/.steam/steam/skins/catppuccin"
+elif [ -d ~/.var/app/com.valvesoftware.Steam ]
+then
+  mkdir -p ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins
+  git clone https://github.com/minischetti/metro-for-steam ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin
+  export METRO="$HOME/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin"
+fi
+ 
+# Install the metro patch
+cd ../ 
+mkdir UPMetro
```

Comment:
a new directory doesn't really need to be made here as `git clone` will already do that. plus, when you use `cd ../` at the end, you will actually return to the original tmp dir

Reply:
First part of your sentence: Good point
Second part of your sentence: That's literally the entire point

Reply:
ah that's my bad, i was thinking of already being in the git cloned directory
you will still need to do that though as right now when you run `git clone`, it will make the `UPMetroSkin`, and when you try to copy `'Unofficial 4.x patch'/'Main Files'/*`, it'll fail since those are in `UPMetroSkin` and not the current directory. you could append a `.` to put it in a new directory, but that would be kinda pointless since we could do this
```bash
cd ..
git clone https://github.com/redsigma/UPMetroSkin
cp -r UPMetroSkin/'Unofficial 4.x patch'/'Main Files'/* "$install_path"
```

Reply:
oh yeah, lemme put that in there real quick :D